src/client/components/rich-text/rich-text.vue

```diff
@@ -1,9 +1,66 @@
 <template>
-  <div v-html="text"></div>
+  <div class="rich-text" v-html="text"></div>
 </template>
 
 <script>
 export default {
   props: ['text']
 }
 </script>
+
+<style>
+.rich-text {
+  width: 100%;
+  max-width: 600px;
+  margin: 0 auto;
+}
+
+.rich-text p {
+  margin-bottom: .8rem;
+  line-height: 1.3;
+}
+
+.rich-text blockquote {
+  position: relative;
+  margin: var(--spacing-double) 0;
+  padding: 1.3rem 0;
+}
+
+.rich-text blockquote:before {
+  content: open-quote;
+  position: absolute;
+  left: 0;
+  top: -35px;
+  font-size: 6rem;
+  font-family: cursive;
+}
+
+.rich-text blockquote p {
+  margin: 0;
+  margin-left: 50px;
+  font-size: 1.2rem;
```

Comment:
Can we use one of the font-size variables for this? To avoid having too many different font-sizes?